Templates/AppSource App/.github/workflows/PullRequestHandler.yaml

```diff
@@ -114,7 +118,7 @@ jobs:
 
   Build:
     needs: [ Initialization ]
-    if: (!failure()) && needs.Initialization.outputs.projectCount > 0
```

Comment:
This fix is also needed in CheckForUpdates.ps1, here:
https://github.com/microsoft/AL-Go/blob/3c2112110a6843827e77458a068579fb18215620/Actions/CheckForUpdates/CheckForUpdates.ps1#L273

and the syntax should be the same as in line 289.
The same syntax should be used above.

If this is not fixed, you will get updates are available immediately after starting

This change should also be done in CICD.yaml, Current.Yaml, NextMinor.yaml and NextMajor.yaml.

Reply:
Still missing the current.yaml workflow